tests/src/JIT/HardwareIntrinsics/X86/Shared/GenerateTests.csx

```diff
@@ -77,212 +77,208 @@ private static readonly (string templateFileName, Dictionary<string, string> tem
 
 private static readonly (string templateFileName, Dictionary<string, string> templateData)[] Sse2Inputs = new []
 {
-    ("SimpleBinOpTest.template",    new Dictionary<string, string> { ["Isa"] = "Sse2", ["LoadIsa"] = "Sse2", ["Method"] = "Add",                                      ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Double", ["Op1VectorType"] = "Vector128", ["Op1BaseType"] = "Double", ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Double",                                          ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetDouble()", ["NextValueOp2"] = "TestLibrary.Generator.GetDouble()", ["ValidateFirstResult"] = "BitConverter.DoubleToInt64Bits(left[0] + right[0]) != BitConverter.DoubleToInt64Bits(result[0])",                                    ["ValidateRemainingResults"] = "BitConverter.DoubleToInt64Bits(left[i] + right[i]) != BitConverter.DoubleToInt64Bits(result[i])"}),
```

Comment:
This PR seems to remove a bunch of the "pack" intrinsics from GenerateTests.csx - was that intentional or a merge error?
Also, the "ignore whitespace" optino on github seems to be broken, so I had to use CodeFlow to see what really changed. In future, it would be great to separate the actual changes from the formatting changes.

Reply:
> This PR seems to remove a bunch of the "pack" intrinsics from GenerateTests.csx 

And the formatting work makes this file unreadable to review.
@4creators could you please just add new tests and make sure no old tests removed?

Reply:
All Pack and Unpack intrinsics will be added in separate PR #19670  which requires rebasing on top of this PR and reimplementation of SseVerify code based on code review. I intend to correct it once this PR is merged.

Reply:
@CarolEidt 
> In future, it would be great to separate the actual changes from the formatting changes.

Sorry for the PR clarity problems. Will try to keep white space changes out of commits with code changes in future.

Hope it will be possible to work with this PR.

Reply:
> All Pack and Unpack intrinsics will be added in separate PR

Why were they removed?

Reply:
I missed that they were included in https://github.com/dotnet/coreclr/pull/19673 before I merged (also due to the combined formatting and changes diff being hard to read).

They were supposed to be part of #19670, which was a separate PR. I left a comment [here](https://github.com/dotnet/coreclr/pull/19673#issuecomment-418864780) indicating that they should be cleaned up, as part of #19670, according to the feedback left there.

They shouldn't be removed as part of this PR.

Reply:
Will correct this error.

Reply:
Correction is in #19866 PR. Waiting to get it merged to rebase on top of it.